libdebug/ptrace/native/libdebug_ptrace_binding.cpp

```diff
@@ -115,6 +115,17 @@ int LibdebugPtraceInterface::prepare_for_run()
     return 0;
 }
 
+Thread& LibdebugPtraceInterface::get_or_raise_exception(const pid_t tid)
```

Comment:
One small thing. To adhere to the expected naming of functions that behave similarly (raising an exception in case of failure), I would suggest renaming the function to "try_get_thread_ref" or "try_get_thread"